java_tools/configuration_definition/src/main/java/com/rusefi/newparse/layout/StructLayout.java

```diff
@@ -4,12 +4,11 @@
 import com.rusefi.newparse.parsing.*;
 
 import java.io.PrintStream;
-import java.lang.reflect.Array;
 import java.util.ArrayList;
 import java.util.List;
 
 public class StructLayout extends Layout {
-    /*private*/public List<Layout> children = new ArrayList<>();
+    /*private*/public final List<Layout> children = new ArrayList<>();
```

Comment:
``/*private*/public`` is that some weird MS style?

Reply:
I think those were intended to be private, but I made them public for some reason...